### ISSUE TITLE
Open for co-maintainership

### ISSUE THREAD

**Issue by chfw** [pyexcel/pyexcel-ezodf#2]
Anyone interested in co-maintaining pyexcel-ezodf? Please reply in this issue. I look forward to working with you.

**Comment by mayanksuman** [comment]
Hi @chfw, I am interested in co-maintenance if project if it is still open for that.

**Comment by chfw** [comment]
yes, it is still open! I thought there is nobody interested hence closed it.

**Comment by mayanksuman** [comment]
> yes, it is still open! I thought there is nobody interested hence closed it.

Then I want to co-maintain the repo. What I should do?

**Comment by jayvdb** [comment]
Happy to help of course.

And I notice @cyberiad packaged it at https://build.opensuse.org/package/show/home:cyberiad/python-ezodf , so they may also be interested.
